### ISSUE TITLE
Update onboarding cards for post-migration

### ISSUE THREAD

**Issue by apbitner** [mozilla-mobile/fenix#6980]
### User Story
As a new user on the new Fenix/Firefox for Android, I want to see onboarding cards that will help me understand the value of key features, so I know the value I will be getting from using this new browser. 

### Dependency
- [ ] Need some solution for Privacy Settings #5905 , but other work could be done.

### Acceptance Criteria

1. [x] New What's new card
2. [x] I can choose whether I want `strict` or `standard` tracking protection (Re-purpose the existing TP card to surface ETP modes as a user selected option)
3. [x] I know that I can choose the position of my nav bar when I first start using the browser. Accessibility users will be defaulted to Top nav bar and we will skip this specific onboarding card for this group.
4. [x] When people click on the FAQ link, it should be similar to what Privacy Policy link does (e.g. "custom tab")
5. [ ] Changes to private browsing copy + adding a link to private browsing settings

**[UX mocks](https://share.goabstract.com/97c30850-4c03-412d-b9b9-495082be6a3a?mode=design&sha=6e74bc5ab42f2d8c069ae917943407ae8dd650d5)**

**Comment by liuche** [comment]
Hey @apbitner, just checking in here - are the onboarding cards for this finalized? or are we still waiting on strings, etc. I already see some mocks for these cards https://app.abstract.com/share/b1b9f719-fbb5-4c94-b45d-556232a3cd52 and just wondering if they are ready for eng! If so, could you move it to the Feature Backlog column (or add an `eng:ready` label)? Thank you!

**Comment by apbitner** [comment]
@liuche I'm reviewing with Vesta & design team today so should be able to wrap them up today/tomorrow. Strings for migration UI (not including these onboarding cards) was finalized last week but I'll get the rest reviewed ASAP once the cards are finalized.

**Comment by mheubusch** [comment]
Copy updates: https://docs.google.com/document/d/1ynhu3K-u4unlHBcf0bjW7PHJcoBBa1UV1-PZfHH1LjM/edit#

**Comment by mcarare** [comment]
@apbitner Does clicking "Open settings" from privacy card finish onboarding?

**Comment by mcarare** [comment]
@apbitner Do we have assets for the toolbar position images?

**Comment by apbitner** [comment]
@mcarare Here are SVG's of the toolbar positions. I've included everything except for the radio button, please let me know if you need something different.

[Illustration - Toolbar Positions.zip](https://github.com/mozilla-mobile/fenix/files/4134461/Illustration.-.Toolbar.Positions.zip)

**Comment by andreicristianpetcu** [comment]
Why does top have minimise, maximise and close like desktop apps? @apbitner
![image](https://user-images.githubusercontent.com/922368/73503397-cd04e800-43d4-11ea-9ef4-3729bce8ef28.png)

**Comment by cadeyrn** [comment]
@andreicristianpetcu These are dummy icons of the Android system bar and not part of Fenix itself...

**Comment by mcarare** [comment]
@apbitner Thank you! I think we would need png's for different configs, like we have for theme selection onboarding card. 
Maybe @AmyYLee can help?  If we don't have these kind of assets already available, I will try to generate them from the svg's.

**Comment by apbitner** [comment]
@mcarare Can you let me know what configs you need exactly? For the theme selection, I believe the assets are the same in both light and dark mode, no?

**Comment by mcarare** [comment]
@apbitner Sorry for not being clearer. I meant screen sizes. For theme onboarding cards we have png for mdpi and for hdpi. ( eg: https://github.com/mozilla-mobile/fenix/blob/master/app/src/main/res/drawable-mdpi/onboarding_dark_theme.png)

**Comment by apbitner** [comment]
@mcarare Ah okay, thanks for clarifying. I usually export SVG but happy to provide PNG's. Here's a zip of both illustrations at mdpi (1x) and hdpi (1.5x). Let me know if you have any issues with them.

[Illustrations PNG.zip](https://github.com/mozilla-mobile/fenix/files/4166404/Illustrations.PNG.zip)

**Comment by sv-ohorvath** [comment]
Verified as fixed on Nightly 2/18. The only issue found: https://github.com/mozilla-mobile/fenix/issues/8481 - not sure if that was intentional, but I couldn't find an issue related to that.
